ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/validator/LedgerStateAccess.scala

```diff
@@ -34,58 +36,74 @@ trait LedgerStateOperations[LogResult] {
 
   /**
     * Reads value of a single key from the backing store.
-    * @param key  key to look up data for
+    *
+    * @param key key to look up data for
     * @return value corresponding to requested key or None in case it does not exist
     */
-  def readState(key: Key): Future[Option[Value]]
+  def readState(key: Key)(implicit executionContext: ExecutionContext): Future[Option[Value]]
```

Comment:
Can you add a wrapper class extending this `LedgerStateOperations` that is compatible with the old API so that updating to the latest DAML SDK can be done with a small change, please? This would be important for Java integrations such as Fabric, Besu, etc. Obviously, we will want to deprecate that helper class later.

Reply:
This should be good to go as `LedgerStateAccess.v1_4` and `LedgerStateOperations.v1_4`. Can you take a look please?

Reply:
Gonna assume you're happy. 🙂